pkg/util/cluster.go

```diff
@@ -353,11 +353,17 @@ func (c *ClusterInfoImp) RefreshDeschedulerCR(ctx context.Context, cl client.Cli
 			}
 			return err
 		}
-		if instance.Spec.ProfileCustomizations == nil {
-			misconfiguredDescheduler = true
+
+		// TODO: remove this once deschedulerv1.RelieveAndMigrate will graduate loosing
+		// the dev prefix
```

Comment:
The TODO is not clear to me. When it says "remove ***this***", what **exactly** "this" refers to?